src/itemObjGen.txt

```diff
@@ -2257,6 +2257,292 @@
     ingredients[]
       name = Rope
       count n = 12
+[]
+  name = Mollusk Walker Body
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood
+      count n = 570
+    ingredients[]
+      name = Rope
+      count n = 12
+    ingredients[]
+      name = Fiber
+      count n = 80
+    ingredients[]
+      name = Fiber Weave
+      count n = 2
+    ingredients[]
+      name = Stone
+      count n = 90
+[]
+  name = Mollusk Walker Legs (1 of 2)
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood
+      count n = 140
+    ingredients[]
+      name = Wood Shaft
+      count n = 18
+[]
+  name = Mollusk Walker Legs Armored (1 of 2)
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood
+      count n = 490
+    ingredients[]
+      name = Wood Shaft
+      count n = 60
+    ingredients[]
+      name = Ceramic Shard
+      count n = 24
+    ingredients[]
+      name = Bone Glue
+      count n = 8
+[]
+  name = Mollusk Walker Wings (1 of 2)
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood
+      count n = 47
+    ingredients[]
+      name = Fiber Weave
+      count n = 28
+[]
+  name = Mollusk Walker Wings Small (1 of 2)
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood Shaft
+      count n = 25
+    ingredients[]
+      name = Rope
+      count n = 12
+    ingredients[]
+      name = Nomad Cloth
+      count n = 2
+[]
+  name = Mollusk Walker Wings Medium (1 of 2)
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood Shaft
+      count n = 50
+    ingredients[]
+      name = Hide
+      count n = 30
+    ingredients[]
+      name = Nomad Cloth
+      count n = 4
+[]
+  name = Mollusk Walker Wings Heavy (1 of 2)
+  category = walkers
+  crafting[]
+    ingredients[]
+      name = Wood Shaft
+      count n = 20
+    ingredients[]
+      name = Rope
+      count n = 9
+    ingredients[]
+      name = Nomad Cloth
+      count n = 6
+[]
+  name = Mollusk Walker Upgrade - Water - Tier 2
+  category = walkers/upgrades
+  crafting[]
+    ingredients[]
+      name = Bone Splinter
+      count n = 16
+    ingredients[]
+      name = Fiber
+      count n = 160
+    ingredients[]
+      name = Hide
+      count n = 24
+    ingredients[]
+      name = Wood
+      count n = 24
+[]
+  name = Mollusk Walker Upgrade - Gear - Tier 2
```

Comment:
we've been going off the columns so this would have been hatch, but i dont know after the patch